specification/cognitiveservices/OpenAI.Inference/models/transcription.create.tsp

```diff
@@ -0,0 +1,95 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Transcription request.")
+model TranscriptionsOptions {
+
+    @doc("The audio file object to transcribe. Binary data.")
+    // how do we handle binary format for a member?
```

Comment:
I think this generalizes to multipart/form input; Gerardo had investigated this a bit early on for authoring and may remember a few tricks, but it's certainly a good topic for the regular meeting.

Reply:
Will resolve the comment for now, since this is a merge blocker anyway for the PR.